go/libraries/doltcore/sqle/logictest/regressions.sql

```diff
@@ -0,0 +1,158 @@
+  `error_message` LONGTEXT,
+  `version` VARCHAR(255) NOT NULL,
+  PRIMARY KEY (`test_file`,`line_num`)
+);
+CREATE TABLE `nightly_dolt_mean_results` (
+  `version` VARCHAR(255) NOT NULL,
+  `test_file` VARCHAR(255) NOT NULL,
+  `line_num` BIGINT NOT NULL,
+  `mean_duration` DOUBLE NOT NULL,
+  `result` LONGTEXT NOT NULL,
+  PRIMARY KEY (`test_file`,`line_num`)
+);
+CREATE TABLE `releases_dolt_results` (
+  `test_file` VARCHAR(255) NOT NULL,
+  `line_num` BIGINT NOT NULL,
+  `duration` BIGINT NOT NULL,
+  `query_string` LONGTEXT NOT NULL,
+  `result` LONGTEXT NOT NULL,
+  `error_message` LONGTEXT,
+  `version` VARCHAR(255) NOT NULL,
+  PRIMARY KEY (`test_file`,`line_num`)
+);
+CREATE TABLE `releases_dolt_mean_results` (
+  `version` VARCHAR(255) NOT NULL,
+  `test_file` VARCHAR(255) NOT NULL,
+  `line_num` BIGINT NOT NULL,
+  `mean_duration` DOUBLE NOT NULL,
+  `result` LONGTEXT NOT NULL,
+  PRIMARY KEY (`test_file`,`line_num`)
+);
+create view `mysql_releases_duration_change`
+as
+select *
+from
+(
+  select 
+  m.test_file, 
+  m.version as mysql_version, 
+  sum(m.mean_duration) as mysql_mean_duration_sum_ms, 
+  r.version as dolt_release_version, 
+  sum(r.mean_duration) as dolt_release_mean_duration_sum_ms, 
+  (100.0 * (cast(sum(m.mean_duration) as decimal(48, 16)) - 
+    cast(sum(r.mean_duration) as decimal(48, 16))) / 
+    (cast((sum(m.mean_duration)) as decimal(48, 16)) + .00001)) as percent_change 
+  from mysql_mean_results as m 
+  join releases_dolt_mean_results as r 
+  on m.test_file = r.test_file and m.line_num = r.line_num
+)
+as wrapped where percent_change < -5.0;
+
+create view `mysql_nightly_duration_change`
+as
+select *
+from
+(
+  select 
+  m.test_file, 
+  m.version as mysql_version, 
+  sum(m.mean_duration) as mysql_mean_duration_sum_ms, 
+  n.version as dolt_nightly_version, 
+  sum(n.mean_duration) as dolt_nightly_mean_duration_sum_ms, 
+  (100.0 * (cast(sum(m.mean_duration) as decimal(48, 16)) - 
+    cast(sum(n.mean_duration) as decimal(48, 16))) / 
+    (cast((sum(m.mean_duration)) as decimal(48, 16)) + .00001)) as percent_change 
+  from mysql_mean_results as m 
+  join nightly_dolt_mean_results as n 
+  on m.test_file = n.test_file and m.line_num = n.line_num
+)
+as wrapped where percent_change < -5.0;
+
+create view `releases_nightly_duration_change`
+as
+select *
+from
+(
+  select 
+  r.test_file, 
+  r.version as release_version, 
+  sum(r.mean_duration) as release_mean_duration_sum_ms, 
+  n.version as nightly_build, 
+  sum(n.mean_duration) as nightly_mean_duration_sum_ms, 
+  (100.0 * (cast(sum(r.mean_duration) as decimal(48, 16)) - 
+    cast(sum(n.mean_duration) as decimal(48, 16))) / 
+    (cast((sum(r.mean_duration)) as decimal(48, 16)) + .00001)) as percent_change 
+  from releases_dolt_mean_results as r 
+  join nightly_dolt_mean_results as n 
+  on r.test_file = n.test_file and r.line_num = n.line_num
+)
+as wrapped where percent_change < -5.0;
+
+create view `mysql_releases_result_change`
```

Comment:
Not sure why we care about these comparisons to mysql results. Mysql is by definition 100% OK (only 99.9% in practice, but close enough). This basically just counts our overall success rate, which is useful, but doesn't need to involve a join. The comparison between release and nightly is more interesting (release = ok, nightly != ok) and useful.